other/install_2-1_mysql.sql

```diff
@@ -1181,7 +1183,7 @@ CREATE TABLE {$db_prefix}mentions (
   content_id INT DEFAULT '0',
   content_type VARCHAR(10) DEFAULT '',
   id_mentioned INT DEFAULT 0,
-  id_member INT(10) UNSIGNED NOT NULL DEFAULT 0,
+  id_member MEDIUMINT(10) UNSIGNED NOT NULL DEFAULT 0,
```

Comment:
the size (10) is here wrong when you use mediumint.

Reply:
See above.   MEDIUMINT does honor display width.